src/samplers/remote_sampler.js

```diff
@@ -118,7 +122,7 @@ export default class RemoteControlledSampler implements Sampler {
       this._logger.error(`Error in fetching sampling strategy: ${err}.`);
       this._metrics.samplerQueryFailure.increment(1);
     };
-    Utils.httpGet(this._host, this._port, `/sampling?service=${serviceName}`, success, error);
+    Utils.httpGet(this._host, this._port, `${this._samplingPath}?service=${serviceName}`, success, error);
```

Comment:
+1

but fwiw, this is soooo broken - what if service name has a space in it? We should be urlencoding URL params.